assets/js/util/index.js

```diff
@@ -573,28 +577,33 @@ export const decodeHTMLEntity = ( str ) => {
  * Get the icon for a module.
  *
  * @param {string}  module                The module slug.
- * @param {boolean} blockedByParentModule Whether the module is blocked by a parent module.
  * @param {string}  width                 The icon width.
  * @param {string}  height                The icon height.
  * @param {string}  useClass              Class string to use for icon.
  *
  * @return {HTMLImageElement}             <img> tag with module icon.
  */
-export function moduleIcon( module, blockedByParentModule, width = '33', height = '33', useClass = '' ) {
+export function moduleIcon( module, width = '33', height = '33', useClass = '' ) {
 	if ( ! global._googlesitekitLegacyData ) {
 		return;
 	}
 
-	/* Set module icons. Page Speed Insights is a special case because only a .png is available. */
-	let iconComponent = <SvgIcon id={ module } width={ width } height={ height } className={ useClass } />;
-
-	if ( blockedByParentModule ) {
-		iconComponent = <SvgIcon id={ `${ module }-disabled` } width={ width } height={ height } className={ useClass } />;
-	} else if ( 'pagespeed-insights' === module ) {
-		iconComponent = <img src={ global._googlesitekitLegacyData.admin.assetsRoot + 'images/icon-pagespeed.png' } width={ width } alt="" className={ useClass } />;
+	switch ( module ) {
+		case 'adsense':
+			return <AdSenseIcon width={ width } height={ height } className={ useClass } />;
+		case 'analytics':
+			return <AnalyticsIcon width={ width } height={ height } className={ useClass } />;
+		case 'optimize':
+			return <OptimizeIcon width={ width } height={ height } className={ useClass } />;
+		case 'pagespeed-insights':
+			return <PageSpeedInsightsIcon width={ undefined } height={ height } className={ useClass } />;
```

Comment:
Width is no longer needed, it originally fixed a visual regression test but a later change fixed it elsewhere. Using a map vs a switch statement (since all props are equal) is a good refactor, making that change as well as returning `null` vs throwing.